### DISCUSSION TITLE
Lualine Git branch: can you keep it viewable inside nvim-tree?

### DISCUSSION THREAD

**Discussion by ScumbagScalawag** [nvim-tree/nvim-tree.lua#2422] [Q&A]
its very minor, but (LunarVim's) Lualine shows `NvimTree_1 [-]` instead of the branch name when focused on nvim-tree. 

Its a little disorienting because my eyes want to always go to the same place to look for what branch I'm on no matter what. 

Thanks

**Comment by alex-courtis** [comment]
lualine is aware of nvim-tree. See `:help lualine-Available-extensions`

You can customise the extension to suit your use case. Maybe just a section A with branch.
